ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/type/TypeMapper.java

```diff
@@ -56,4 +56,12 @@ static void setDefaultValue(Schema schema, Object defaultValue) {
         }
         schema.setDefault(defaultValue);
     }
+
+    SchemaResult getSchema(TypeSymbol typeSymbol, boolean enableExpansion) throws UnsupportedOperationException;
+
+    Schema getSchema(TypeSymbol typeSymbol) throws UnsupportedOperationException;
+
+    String getJsonSchemaString(TypeSymbol typeSymbol) throws UnsupportedOperationException;
```

Comment:
Shall we add tests for this new API?

Reply:
At the moment we don't have unit tests for this package. Thinking of addressing this in a separate issue.

Reply:
Writing a compiler plugin just to test this API seems to be not efficient approach. Since this is for an internal requirement and the package which uses this API has tests, we can ignore the testing part for now.

@MohamedSabthar we have tests from the agent library right?

Reply:
Yes, we are currently adding few test cases.